sections/semantics-embedded-content.include

```diff
@@ -597,6 +598,7 @@
           attribute DOMString sizes;
           attribute DOMString? crossOrigin;
           attribute DOMString useMap;
+          attribute DOMString longDesc; // defined in [[!LONGDESC]]
```

Comment:
The [[!LONGDESC]] reference here doesn't link to anything after bikeshed runs, probably cause it's in a comment in the IDL code. Not sure if this can be fixed.